test/unit/qme/map/measure_calculation_job_test.rb

```diff
@@ -48,8 +48,7 @@ def test_perform_recalculate
   def test_perform_no_recalculate
     options = {
       'measure_id' => '2E679CD2-3FEC-4A75-A75A-61403E5EFEE8',
-      'effective_date' => Time.gm(2011, 1, 15).to_i,
-      'status.state' => 'complete'
```

Comment:
note: deleted this line as it wasn't affecting the tests on bump_mongoid branch that we're currently using. For instance, the status' state is registered as 'unknown' rather than 'complete' or 'completed.' evergreen test essentially...